tests/regression/astdumptest/allexpressions-expected.ast

```diff
@@ -1,15 +1,24 @@
+//Parameter("")
```

Comment:
Do we really want to add empty annotations to every assignment, and dump it here?

Reply:
That's what I thought first too and removed that output, but that failed all the customizer tests. I think those would be rendered pretty useless with that simple change.
E.g. https://github.com/openscad/openscad/blob/master/tests/regression/customizertest/parameter-expected.ast
So I reverted that and just regenerated that base AST tests. I guess we need a better way of testing the customizer parser so we can get rid of the sometimes useless and also non-conforming (I mean export is different from import) test cases.

Reply:
perhaps suppressing output of empty parameter annotations, and adjusting test cases accordingly would suffice?

Reply:
I don't see a good way that is not adding horrible code just to optimize for some nicer test output. It might make more sense to see why we are generating empty annotations in the first place. But that's something I'd rather not touch at RC2 time.

Reply:
That's a fair assessment; we can fix this later (or sooner if Michael deems it an innocent fix)